### DISCUSSION TITLE
不同云账号有不同的jumpserver管理账户的场景能否支持？

### DISCUSSION THREAD

**Discussion by starslcn** [starsliao/ConsulManager#28] [Q&A] [answered]
不同云账号有不同的jumpserver管理账户的场景能否支持？

**Comment by starsliao** [comment, marked as answer]
### 目前web界面上不支持这样的场景，不过后端是已经支持的，所以可以直接修改consul KV的方式来实现。
- 访问consul的webUI `http://x.x.x.x:8500/ui/dc1/kv/ConsulManager/jms/`
- 该目录下可以看到2个键：全局管理用户信息：`ecs_info`，全局特殊管理用户信息：`custom_ecs_info`
- 进入改目录下对应的云厂商以及云账户的目录
- 把上面提到了两个键复制到云账户的目录下即可，并修改为需要的内容即可，注意内容的格式保持不变。
- 下次同步时候会优先读取云账户目录下的管理用户信息。（需要登录jumpserver删掉已同步的主机。）
- 我会尽快把这个功能做到web界面上。

**Reply by wudongyu** [reply]
nice
